app/const.py

```diff
@@ -18,6 +18,9 @@
 # We use lazy translation in this file because the language isn't set yet.
 from django_setup import gettext_lazy as _
 
+# The index name
+PERSON_FULLTEXT_INDEX_NAME = 'person_information'
```

Comment:
If this is used only in full_text_search.py, it would be better to define this in full_text_search.py instead of here. As explained at the top of this file, this file is for "Constants that aren't specific to a particular module or handler."

In general, if something is used only in a specific module/class/method, it's better to define it in the module/class/method.